cogs/displayinfo.py

```diff
@@ -118,39 +128,36 @@ async def on_member_update(self, mem_before:discord.Member, mem_after:discord.Me
                         print("end:", self.timers) # printing the dictionary to see the updates after an activity is ended
                         self.current_collection.replace_one({}, self.timers) # Update the MongoDB collection with the updated activity data(self.timers)
 
-    async def update_now(self, string_id):
-        updated = False
-        if string_id in self.timers and self.current_date in self.timers[string_id]:
-            for activity, info in self.timers[string_id][self.current_date].items():
-                if info["startTime"] != None:
-                    updated = True
-                    currentStartTime = self.timers[string_id][self.current_date][activity]["startTime"]
-                    playTime = datetime.now() - currentStartTime 
-                    self.timers[string_id][self.current_date][activity]["totalTime"] = self.timers[string_id][self.current_date][activity].get("totalTime", 0) + playTime.total_seconds()
-                    self.timers[string_id][self.current_date][activity]["startTime"] = datetime.now()
-        return updated
-
+    #This function periodically updates the activities of members that are online and in an activity.
     async def timed_update(self):
```

Comment:
All functions that are not based on discord.py or called as a command (used as helper functions) can be marked with the `_` prefix such as `_timed_update()` and `_reset_start()`. Let's also put all of these functions near the top to just have them grouped together for easier readability.